feat #148 Creating the mobile app project

onurkck928 opened the pull request:
Added the initial mobile app project created with React Native. Fixes #148,  contents of the mobile app should be built on this inital setup.

orbitstabilizer reviewed (approved):
I think, we can add native builds to `.gitignore`:
```
ios 
android
```
Nit pick:
- Web frontend calls the app `nutrihub`, we may also maintain that naming

onurkck928 commented:
> I have run the setup locally and it works fine on my machine (ios, android) I think, we can add native builds to `.gitignore`:
> 
> ```
> ios 
> android
> ```
> 
> Nit pick:
> 
> * Web frontend calls the app `nutrihub`, we may also maintain that naming

I have modified the gitignore and changed the app name in the latest commit according to your suggestion

orbitstabilizer commented:
Thanks for your efforts, lets merge this!
